### ISSUE TITLE
Model produces 0 loss and has very less no. of parameters

### ISSUE THREAD

**Issue by neel04** [keras-team/autokeras#1467]
### Setup Details
Include the details about the versions of:
 - OS type and version: Ubuntu - **Colab PRO**
 - Python: 3.x
 - autokeras: master
 - keras-tuner: master
 - tensorflow: 2.3.0

I had a Pandas DataFrame of about 600Mb. However, when I have trained the model, the "best" model size comes out to be of `2,433` parameters whereas the expectation was that it would run in millions, if not billions. Also, during training, I was noticing that the `val_loss` was reducing substantially but when I evaluate the model on the training dataset, it produces `loss:0.000e+0` and same for the `mse`.

Can anyone help me in figuring out why the model doesn't perform as well as it seemed to in training? Can I place a value for the minimum number of parameters? Could it be because `AutoKeras` can't handle this much amount of data?

**Comment by haifeng-jin** [comment]
It is mainly caused by AutoKeras not covering the suitable models for your dataset.
If you can send us your dataset, we can take some time to inspect the reasons.
